app/src/main/java/org/mozilla/rocket/privately/PrivateTabViewProvider.kt

```diff
@@ -12,6 +12,10 @@ class PrivateTabViewProvider(private val host: Activity) : TabViewProvider {
         return WebViewProvider.create(host, null, WebViewSettingsHook) as TabView
     }
 
+    override fun purify() {
+        create().cleanup()
```

Comment:
Does this create a new Tabview every time a client call purify()?

Reply:
yes, why?

Reply:
Nevin was worrying about performance waste. We will refactor `TabView.cleanup()` soon it should be OK for now.